tpkg/cutest/cutest_namedb.c

```diff
@@ -86,14 +84,13 @@ create_and_read_db(CuTest* tc, region_type* region, const char* zonename,
 
 	/* read the db */
 	memset(&nsd, 0, sizeof(nsd));
-	nsd.db = db = namedb_open(dbfile, opt);
+	nsd.db = db = namedb_open(opt);
 	if(!db) {
-		printf("failed to open %s: %s\n", dbfile, strerror(errno));
+		printf("failed to open namedb: %s\n", strerror(errno));
```

Comment:
I do not think that `namedb_open` returns an error, such that the printout of `errno` is helpful. This is likely because the `mmap()` call is removed from it.
```suggestion
		printf("failed to open namedb\n");
```

Reply:
Fixed.